test/Microsoft.AspNetCore.Mvc.Razor.Test/TagHelpers/TagHelperComponentTagHelperTest.cs

```diff
@@ -282,6 +304,45 @@ public void Init_LogsTagHelperComponentInitialized()
             Assert.Equal($"Tag helper component '{typeof(TestTagHelperComponent)}' processed.", sink.Writes[0].State.ToString(), StringComparer.Ordinal);
         }
 
+        [Fact]
+        public void Init_InitializesTagHelperComponent_WithViewContext()
```

Comment:
Hmm, in this case Init doesn't set view context. It tries to query request service for the activator. You'll have to change this test around a bit.